bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -55,6 +55,11 @@
   {# L10n: This is the string used to indicate that the plugin is unknown #}
   data-unknown="{{ _('unknown') }}"
 
+  {% if l10n_has_tag('plugin_outdated') -%}
+    {# L10n: This is the string used to indicate that the plugin is outdated #}
+    data-outdated="{{ _('outdated') }}"
+  {% endif %}
```

Comment:
How often are we going to show this string? If it's something we risk to display often, I think we need to live with either the confusing "vulnerable", or maybe "unknown".